spec/cassettes/check_benefits_requests/GET_/providers/applications/_application_id/check_benefits/when_the_check_benefit_result_already_exists/and_the_applicant_has_since_been_modified/generates_a_new_check_benefit_result.yml

```diff
@@ -0,0 +1,199 @@
+---
```

Comment:
I think this file might not be needed. Maybe you rename the test and it generated another cassette but left this one?

Reply:
There was an extra cassette that I've now removed.